third_party/move/move-compiler/tests/move_check/v2-not-supported/index.exp

```diff
@@ -0,0 +1,102 @@
+warning[W09001]: unused alias
```

Comment:
Please move this test into `tests/checking-lang-v1` in the compiler v2 directory. Should not add new tests to compiler v1 tests, also other tests already there.

Reply:
Added the test to tests/checking-lang-v1